### ISSUE TITLE
Add Consultant Save 

### ISSUE THREAD

**Issue by mounikaalladi** [acsestech/virtualassociate#15]
![image](https://cloud.githubusercontent.com/assets/23167576/19888925/cf05299a-a057-11e6-83a2-2a07caa080b8.png)

While i tried to save record without entering any data it displaying as select date of birth Proper message should be displayed.

**Comment by sandipkumarnandi** [comment]
Every validation Item are taken one at a time, so going by that if DOB is left blank it pops that one, and after thatn any other item if any will popup if left blank, can this be considered as bug ?

**Comment by bhargavibindhu** [comment]
I am not able to add new consultants now.

![image](https://cloud.githubusercontent.com/assets/20394163/21493877/d54da068-cc33-11e6-8630-bbd3a6bc93fb.png)

**Comment by bhargavibindhu** [comment]
Hari Krishna the consultant http://va.antronsys.com/VAModule/Consultant/AddEditConsultants/88
is mapped to the client Sensiple.
![image](https://cloud.githubusercontent.com/assets/20394163/21522135/0a4991b4-cd29-11e6-91cb-ab68ae43332b.png)

But he is not assigned in the Task assignment list 
![image](https://cloud.githubusercontent.com/assets/20394163/21522146/1ed8a016-cd29-11e6-8fa7-06891952deb4.png)

Only after checking this flow, i can go with other bugs.

**Comment by NaziniSK** [comment]
please assign any project to  default client employee of sensiple ,as I assigned test32  (project) for default Client Employee of sensiple( refer below 2nd image) , also we need to add the same project (test 32) for consultant (Hari Krishna) in project consultant  refer(third image).
now add new task assignment (refer fourth image)-> select test32 project -> in assigned to you will get this consultant(Hari Krishna)

![image](https://cloud.githubusercontent.com/assets/23185953/21539892/e33fd97c-cdd0-11e6-86c0-2cae7cdcfdd4.png)

![image](https://cloud.githubusercontent.com/assets/23185953/21539911/23e2e938-cdd1-11e6-829b-d21a6fcfd7aa.png)
![project consultant](https://cloud.githubusercontent.com/assets/23185953/21539944/97c53ab8-cdd1-11e6-8c8d-a9cddfb53e13.png)
![image](https://cloud.githubusercontent.com/assets/23185953/21540000/0f6c0600-cdd2-11e6-9204-f5c0f7c00332.png)

**Comment by NaziniSK** [comment]
only client employee who has same project which is associated in project consultant can assign task to that 
consultant .
![image](https://cloud.githubusercontent.com/assets/23185953/21540103/e1fd99da-cdd2-11e6-8cae-4015e45a9bd2.png)
